src/appleseed.studio/mainwindow/project/projectexplorer.cpp

```diff
@@ -247,6 +253,20 @@ void ProjectExplorer::slot_edit_item(QTreeWidgetItem* item, int column)
     static_cast<ItemBase*>(item)->slot_edit();
 }
 
+void ProjectExplorer::slot_drag_item(QTreeWidgetItem* item, int column)
+{
+    if (item && item->flags() & Qt::ItemIsDragEnabled)
+    {
+        QDrag* drag = new QDrag(m_tree_widget);
```

Comment:
The code might be correct, but who owns the allocated memory? When is it released?

Reply:
I dont know, I followed the example provided by qt but there is nothing provided about the memory management of QDrag
And it seems to be the same for the menu (droppedmaterialmenu)